plugins/outputs/prometheus_client/README.md

```diff
@@ -10,6 +11,14 @@ This plugin starts a [Prometheus](https://prometheus.io/) Client, it exposes all
   ## Address to listen on.
   listen = ":9273"
 
+  ## Metric version controls the mapping from Telegraf metrics into
+  ## Prometheus format.  When using the prometheus input, use the same value in
+  ## both plugins to ensure metrics are round-tripped without modification.
+  ##
+  ##   example: metric_version = 1; deprecated in 1.13
+  ##            metric_version = 2
```

Comment:
Other similar comments have a note that version 2 is the "recommended version".  Not sure why it's missing here

Reply:
I added the missing text.